SudoScript.Core.Test/BoardTests.cs

```diff
@@ -0,0 +1,120 @@
+﻿using NUnit.Framework;
+using SudoScript.Core;
+using SudoScript.Core.Data;
+
+namespace SudoScript.Core.Test;
+
+internal sealed class BoardTests
+{
+    [Test]
+    public void ValidateBoardTest()
+    {
+        Board board = Util.CreateStandardEmpty();
+        board[5, 1].Digit = 2;
+        board[4, 8].Digit = 7;
+        board[5, 2].Digit = 2;
+        board[8, 4].Digit = 4;
+
+        Assert.That(board.Validate(), Is.False);
+        board[5, 2].Digit = 3;
+        Assert.That(board.Validate(), Is.True);
+
+        board[9, 9].EliminateCandidate(1);
+        board[9, 9].EliminateCandidate(2);
+        board[9, 9].EliminateCandidate(3);
+        board[9, 9].EliminateCandidate(4);
+        board[9, 9].EliminateCandidate(5);
+        board[9, 9].EliminateCandidate(6);
+        board[9, 9].EliminateCandidate(7);
+        board[9, 9].EliminateCandidate(8);
+        board[9, 9].EliminateCandidate(9);
+
+        Assert.That(board.Validate(), Is.False);
+    }
```

Comment:
This is multiple different unit tests in one. This is the reason we usually use AAA for unit tests (arrange act assert) which this unit test definitely does not follow.

Reply:
Addressed with [5a73406]